src/lib.rs

```diff
@@ -1224,6 +1226,8 @@ impl TestKitApi {
             resp.expect("Got unexpected `Err(..)` response")
         };
 
+        debug!("GET request: {} Response: {:?}", url, resp);
```

Comment:
Nitpick: I'd use the original URL (possibly preceded by a slash) here and in POST request debugging to avoid questions like "where does `http://localhost:3000/` part come from".

It also could be nice to debug whether the request belongs to public or private API, I suppose; this probably requires adding an additional param to `get_internal` and `post_internal` (or replacing the `&Mount` param), say, an enum with variants `TestKitApiHandler::Private` and `TestKitApiHandler::Public`.

In summary, the output would look like

```none
Private GET /api/services/cryptocurrency/wallets
Response: [
  ...
]
```